src/price_provider/gofer/mod.rs

```diff
@@ -1,30 +1,52 @@
+use crate::price_provider::gofer::types::GoferPriceRequest;
 use crate::price_provider::{Price, PriceProvider, PRECISION_FACTOR};
+use async_trait::async_trait;
 use eyre::Result;
-use std::process::Command;
 
 mod types;
 
 pub struct GoferPriceProvider {
-    gofer_cmd: String,
+    gofer_url: String,
+    pair: String,
 }
 
 impl GoferPriceProvider {
-    pub fn new(gofer_cmd: &str) -> GoferPriceProvider {
+    pub fn new(gofer_url: &str, pair: String) -> GoferPriceProvider {
```

Comment:
Not sure if we really need to make the pair configurable or can keep it hardcoded. 
As far as I know we only plan to submit ETH/USD prices for now. 
@alextes What do you think ?

Reply:
Yea I think we can keep the ETHUSD pair hardcoded for the foreseeable future. Although I'd be open to having `new` use a hardcoded pair for example but having a second constructor which does allow any other pair to be passed if that helps @konstantinzolotarev in any way.